apps/rush-lib/package.json

```diff
@@ -18,7 +18,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@aws-sdk/client-s3": "~3.3.0",
```

Comment:
Nice!  I'm impressed we were able to eliminate this huge dependency with such a relatively small amount of custom code. 😎